src/main/java/Duke.java

```diff
@@ -1,10 +1,271 @@
+import dukeExceptionsPackage.*;
+import dukeTasksPackage.*;
+import java.io.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import static java.lang.System.exit;
+
 public class Duke {
+    private static final String COMMAND_BYE = "bye";
+    private static final String MESSAGE_BYE = "Duke: Bye. Hope to see you again soon!";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_MARK = "mark";
+    private static final String COMMAND_UNMARK = "unmark";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_DELETE = "delete";
+    private static final String MESSAGE_DELETE = "Duke: Noted. I have deleted the task below: ";
+    private static final String COMMAND_CLEAR = "clear";
+    private static final String MESSAGE_CLEAR = "file has been cleared";
+
+    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNext()) {
+            System.out.println(s.nextLine());
+        }
+    }
+    private static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+    private static void clearFile(String filePath) throws IOException {
+        FileWriter fw = new FileWriter(filePath, false);
+        PrintWriter pw = new PrintWriter(fw, false);
+        pw.flush();
+        pw.close();
+        fw.close();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String directoryPath = "C:\\Users\\cwxky\\projects\\cs2113-git\\data";
```

Comment:
Perhaps you can consider replacing the path here with a constant variable instead, so as to eliminate magic literals in your code.

Reply:
Additionally, you may also want to reconsider whether this would result in bugs. Consider the case of other users of the app, will they have a similar directory structure (EG macos/linux devices).